user/TASK/chassis_task/chassis_task.c

```diff
@@ -199,12 +211,19 @@ char pid_out[64];
  * @retval None
  */
 static void increment_PID(Chassis_t *chassis_pid){
-	//translation
+    
+    for(int i = 0; i < 4; i++){
+        chassis_pid->motor[i].current_out = chassis_pid->motor[i].speed_set;
+    }
```

Comment:
Comment: There should be a comment here explaining why this is done

Question: Does this lead to smoother driving (determined by testing)? Wouldn't it be better to just increment/decrement current_out (as lines below already do) instead of setting it to some potentially "far away" value every iteration?

Reply:
Yes, this leads to better driving. The PID POSITION is misleading, it is actually controlling the velocity with the standard PID controller. PID DELTA is when the input for position is actually a change in position from the previous time stamp. Look at pid.c for more information. 
Essentially, PID DELTA should only every be used when dealing with gyro data!